src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,34 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title: postTitle, body: postBody, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{postTitle}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+
+        </p>
```

Comment:
```suggestion
          <UserInfo user={user} />
        </p>
```
Don't leave empty lines between parent and child elements

Reply:
+